pkg/storage/mysql/mysql_test.go

```diff
@@ -25,6 +23,7 @@ func TestMySQLDatastore(t *testing.T) {
 	require.NoError(t, err)
 	defer ds.Close()
 	test.RunAllTests(t, ds)
+	test.ReadAuthorizationModelUnmarshallErrorTest(t, "mysql", uri, ds)
```

Comment:
Instead of adding this here, would you please put it in the `RunAllTests` function so that it gets run in that group of tests?